cabot/cabotapp/models.py

```diff
@@ -559,7 +561,7 @@ def run(self):
         result.save()
 
         # would like to do this in bulk, but django can't fetch the tag object PKs from bulk_create()...
-        tag_objs = [StatusCheckResultTag.objects.get_or_create(value=t)[0] for t in tags]
+        tag_objs = [StatusCheckResultTag.objects.get_or_create(value=t)[0] for t in tags if t]
```

Comment:
If someone adds `None` or `""` as a tag by accident, this will raise an `IntegrityError` - so I filtered it out with `if t`.